service/src/main/java/bio/terra/catalog/service/DatasetService.java

```diff
@@ -181,6 +187,39 @@ private DatasetPreviewTable generateDatarepoTable(
                 .getResult());
   }
 
+  private DatasetPreviewTable generateRawlsTable(
+      AuthenticatedUserRequest user, Dataset dataset, String tableName) {
+    Map<String, EntityTypeMetadata> ent =
+        rawlsService.entityMetadata(user, dataset.storageSourceId());
+    EntityTypeMetadata obj = ent.get(tableName);
+    return new DatasetPreviewTable()
+        .columns(convertEntityToColumn(obj))
+        .rows(
+            rawlsService
+                .entityQuery(user, dataset.storageSourceId(), tableName)
+                .getResults()
+                .stream()
+                .map(Entity -> convertEntityToRow(Entity, obj.getIdName()))
```

Comment:
In general variable names start with a lower case letter, so this should be `entity` not `Entity`
```suggestion
                .map(entity -> convertEntityToRow(entity, obj.getIdName()))
```